src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -24,10 +38,14 @@ import {
   Path,
   Runner
 } from 'src/types'
+import Dialog from '../../../../components/UI/Dialog'
+import DialogContent from '../../../../components/UI/Dialog/DialogContent'
```

Comment:
I believe you could have an index.ts file that imports all `Dialog` children's (like we are doing with sidebar, for instance) and then just import all of them on the same line:
`import {DialogContent, DialogFooter, DialogHeader} from 'src/components/UI/Dialog/components'`